### ISSUE TITLE
Handle DNA atom records in PDBs

### ISSUE THREAD

**Issue by tonyelewis** [UCLOrengoGroup/cath-tools#23]
Example: 10mh

**Comment by tonyelewis** [comment]
Example errors for `10mh` : 

~~~~~
[2017-01-17 15:14:46.213809] [0x00002b6e8a5f36c0] [warning] Skipping PDB atom record "ATOM      1  P    DC B 402     -21.395  27.953 101.986  1.00 57.02           P  " with message: Do not recognise amino acid entry: " DC" - ../source/structure/protein/amino_acid.cpp(136): Throw in function void cath::amino_acid::set_letter_code_or_name(const string&)
Dynamic exception type: boost::exception_detail::clone_impl<cath::common::invalid_argument_exception>
std::exception::what: Amino acid name is actually a DNA/RNA base ("DC") rather than a valid amino acid

[2017-01-17 15:14:46.213886] [0x00002b6e8a5f36c0] [warning] Skipping PDB atom record "ATOM      2  OP1  DC B 402     -20.559  27.880 103.241  1.00 57.56           O  " with message: Do not recognise amino acid entry: " DC" - ../source/structure/protein/amino_acid.cpp(136): Throw in function void cath::amino_acid::set_letter_code_or_name(const string&)
Dynamic exception type: boost::exception_detail::clone_impl<cath::common::invalid_argument_exception>
std::exception::what: Amino acid name is actually a DNA/RNA base ("DC") rather than a valid amino acid

[2017-01-17 15:14:46.213917] [0x00002b6e8a5f36c0] [warning] Skipping PDB atom record "ATOM      3  OP2  DC B 402     -21.971  29.249 101.499  1.00 55.84           O  " with message: Do not recognise amino acid entry: " DC" - ../source/structure/protein/amino_acid.cpp(136): Throw in function void cath::amino_acid::set_letter_code_or_name(const string&)
Dynamic exception type: boost::exception_detail::clone_impl<cath::common::invalid_argument_exception>
std::exception::what: Amino acid name is actually a DNA/RNA base ("DC") rather than a valid amino acid
[...]
~~~~~

**Comment by tonyelewis** [comment]
Possible values are: `A`, `C`, `G`, `N`, `U`, `DA`, `DC`, `DG`, `DT` and `DU`

**Comment by tonyelewis** [comment]
Fixed in 23e7fab988afb408ae262de762c8db8dc0c99681
